client/src/Pages/Login/index.js

```diff
@@ -1,11 +1,108 @@
-import React from 'react'
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import PersonIcon from '@material-ui/icons/Person';
+import { makeStyles } from '@material-ui/core';
+import Axios from 'axios';
+import Container from '../../components/Container';
+import Button from '../../components/Button';
+import InputFeild from '../../components/InputFeild';
+import validationLogin from '../../Utils/validationLogin';
+import Home from '../Home';
+
+const useStyle = makeStyles((them) => ({
+  root: {
+    margin: them.spacing(1),
+  },
+  input: {
+    width: '60%',
+  },
+  loginIcon: {
+    margin: them.spacing(2),
+  },
+}));
 
 function LoginPage() {
-    return (
-        <div>
-           <h1>Login Page</h1> 
-        </div>
-    )
+  const classes = useStyle();
+  const history = useHistory();
+  const [data, setData] = useState({});
+  const handleChange = (e) => {
+    setData((prev) => ({ ...prev, [e.target.name]: e.target.value }));
+  };
+
+  console.log(data);
+
+  const handlePush = async (e) => {
+    try {
+      e.preventDefault();
+      await validationLogin.validate(data);
+      await Axios.post('https://jsonplaceholder.typicode.com/posts', data)
+        .then((res) => {
+          console.log(res);
+        })
+        .catch((err) => {
```

Comment:
I'm not 100% sure, but I think you can remove this catch statement and then any error will bubble out to your other catch statement below (line 48).  Then you could have just one place to handle both types of errors, which would be a little simpler for you.